.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+    - master
+    - main
+    - develop
+
+jobs:
+  pre-commit:
+    name: Format
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
```suggestion
    - uses: actions/checkout@v2
       with:
         submodules: recursive
```

Reply:
Actually, Fastjet has a submodule within it, for SISCone, so I don't want to limit the recursive depth.

Reply:
There are only two options. `submodules: true` checkout out one level, and `submodules: recursive` checks out all levels. `fetch-depth` refers to the git history, not submodules.